web/libs/datamanager/src/components/DataManager/Toolbar/ActionsButton.js

```diff
@@ -45,14 +45,14 @@ export const ActionsButton = injector(observer(({ store, size, hasSelected, ...r
   const actions = store.availableActions
     .filter((a) => !a.hidden)
     .sort((a, b) => a.order - b.order);
-
+  
   const invokeAction = (action, destructive) => {
     if (action.dialog) {
-      const { type: dialogType, text, form } = action.dialog;
+      const { type: dialogType, text, form, title, modal_title } = action.dialog;
```

Comment:
In the below definition of the ActionDialog model it is missing `title`. Do we need both `title` and `modal_title`?

If not, we can collapse this down to:

```suggestion
      const { type: dialogType, text, form, title } = action.dialog;
```

If we do require both I'd suggest camelCase'ing the modal_title to be locally consistent:

```suggestion
      const { type: dialogType, text, form, title, modalTitle } = action.dialog;
```